.github/workflows/universalnrp-test.yml

```diff
@@ -11,7 +14,18 @@ jobs:
       matrix:
         target:
           [
+            { os: centos, version: 8, package-type: RPM, tag: ''},
+            { os: debian, version: 10, package-type: DEB, tag: ''},
+            { os: debian, version: 11, package-type: DEB, tag: ''},
+            # { os: debian, version: 12, package-type: DEB, tag: ''},
```

Comment:
># { os: debian, version: 12, package-type: DEB, tag: ''}, [](http://example.com/codeflow?start=12&length=57)

Why are Debian 12 and Rocky Linux 9 disabled here?

Reply:
Although the images are created, they have issues booting. Im Fixing this and left them commented out for now

Reply:
We also need Amazon Linux added.

Reply:
Removed distros with issues in automation. Will have new PR with them reintroduced when working.